mitodl/hugo-course-publisher
# Video Gallery

#### Pre-Flight checklist


#### What are the relevant tickets?
Closes https://github.com/mitodl/hugo-course-publisher/issues/120

#### What's this PR do?
This PR adds a layout for `videogallery` pages that iterates children of the page with the `video` layout and generates cards with links to the various videos.

#### How should this be manually tested?
Run the site, view a page with a video gallery like http://localhost:3000/courses/15-s21-nuts-and-bolts-of-business-plans-january-iap-2014/sections/lecture-videos and make sure it renders properly and the links work on both mobile and desktop.

#### Screenshots (if appropriate)
![Screen Shot 2020-06-23 at 2 05 12 PM](https://user-images.githubusercontent.com/12089658/85440070-aa2dc880-b55b-11ea-9252-fd4accdd7189.png)

![Screen Shot 2020-06-23 at 2 05 29 PM](https://user-images.githubusercontent.com/12089658/85440084-adc14f80-b55b-11ea-8045-75f217bf9e75.png)
